contributing/code/bugs.rst

```diff
@@ -14,8 +14,8 @@ Before submitting a bug:
 * Double-check the official :doc:`documentation </index>` to see if you're not misusing the
   framework;
 
-* Ask for assistance on the `users mailing-list`_, the `forum`_, or on the
-  #symfony `IRC channel`_ if you're not sure if your issue is really a bug.
+* Ask for assistance on `Stack Overflow`_ or on the #symfony `IRC channel`_
```

Comment:
Could we change the `IRC channel` link to point to `http://symfony.com/irc`? I find this page ore humand-friendly and useful than linking to the `irc://` protocol.